ui-v2/app/components/topology-metrics/series/index.hbs

```diff
@@ -1,14 +1,11 @@
-<DataSource
-  @src={{uri nspace dc 'metrics' 'summary-for-service' @service @protocol}}
-  @onchange={{action 'change'}} />
-
-{{on-window 'resize' (action 'redraw')}}
-
-<div class="sparkline-wrapper">
-  <div class="tooltip">
-    <div class="sparkline-time">Timestamp</div>
+{{#if (not error)}}
+  {{on-window 'resize' (action 'redraw')}}
+  <div class="sparkline-wrapper" {{did-update (action "change") @data}}>
+    <div class="tooltip">
+      <div class="sparkline-time">Timestamp</div>
+    </div>
+    <div class="sparkline-loader"><span>Loading Metrics</span></div>
+    <svg class="sparkline"></svg>
   </div>
-  <div class="sparkline-loader"><span>Loading Metrics</span></div>
-  <svg class="sparkline"></svg>
-</div>
+{{/if}}
```

Comment:
In the error case does it just show blank now? It would be nicer if we showed some sort of error I guess? Before this PR the series/index.js would at least show a message like "No Data Available". Could we do the same until we have a cleaner error state to show?